src/ratchet.rs

```diff
@@ -1,11 +1,10 @@
-use std::fmt::{self, Display, Formatter};
-
 use crate::{
     constants::{LARGE_EPOCH_LENGTH, MEDIUM_EPOCH_LENGTH, RATCHET_SIGNIFIER},
     hash::Hash,
     PreviousErr, RatchetErr,
 };
 use rand::Rng;
+use std::fmt::{self, Display, Formatter};
```

Comment:
Is there a reason for choosing this order over having `std` stuff first?

Reply:
essentially, it's all abc order for everything, and most editors tend to format that way too.